src/components/Display.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Display extends React.Component {
+  render() {
+    return <h1>{this.props.result}</h1>;
+  }
+}
+
```

Comment:
`result` should be string (enforce with `propTypes`) [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)]